### PR TITLE
Creates links to new resources for InputLookupSinopia Components

### LINKED ISSUE #1731: Links to create new resources from InputLookupSinopia
image

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #1731

### CODE REVIEW

File: __tests__/components/editor/property/ResourceList.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import { wait } from '@testing-library/react'
+import ResultList from 'components/editor/property/ResourceList'
+import * as sinopiaSearch from 'sinopiaSearch'
+import {
+  renderWithReduxAndRouter, createReduxStore, createBlankState,
+} from 'testUtils'
+
+jest.mock('sinopiaSearch')
+/* eslint import/namespace: 'off' */
+sinopiaSearch.getTemplateSearchResults = jest.fn().mockReturnValue(Promise.resolve(
```
Comment (justinlittman) [inline __tests__/components/editor/property/ResourceList.test.js]: Could just be `mockResolvedValue`.

File: __tests__/components/editor/property/ResourceList.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import React from 'react'
+import { wait } from '@testing-library/react'
+import ResultList from 'components/editor/property/ResourceList'
+import * as sinopiaSearch from 'sinopiaSearch'
+import {
+  renderWithReduxAndRouter, createReduxStore, createBlankState,
+} from 'testUtils'
+
+jest.mock('sinopiaSearch')
+/* eslint import/namespace: 'off' */
```
Comment (justinlittman) [inline __tests__/components/editor/property/ResourceList.test.js]: `// eslint-disable-next-line import/namespace`?

File: src/components/editor/property/ResourceList.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import { useDispatch, useSelector } from 'react-redux'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { getTemplateSearchResults } from 'sinopiaSearch'
+import shortid from 'shortid'
+import { newResource } from 'actionCreators/resources'
+import { findAuthorityConfigs } from 'utilities/authorityConfig'
+import { getPropertyTemplate } from 'selectors/resourceSelectors'
+
+export const newResourceErrorKey = 'newresource'
```
Comment (justinlittman) [inline src/components/editor/property/ResourceList.jsx]: This error is not surfaced to users. Needs an `<Alerts>` added to ResourceTemplate.jsx referencing this error key.

File: __tests__/components/editor/property/ResourceList.test.js (hunk 1 above)
Comment (jermnelson) [reply __tests__/components/editor/property/ResourceList.test.js]: Fixed in latest commit

File: __tests__/components/editor/property/ResourceList.test.js (hunk 2 above)
Comment (jermnelson) [reply __tests__/components/editor/property/ResourceList.test.js]: Changed in latest commit

File: src/components/editor/property/ResourceList.jsx (hunk 3 above)
Comment (jermnelson) [reply src/components/editor/property/ResourceList.jsx]: Fixed in latest commit.

File: src/components/editor/ResourceTemplate.jsx [hunk 4]
```diff
@@ -31,6 +31,7 @@ class ResourceTemplate extends Component {
       <div className="ResourceTemplate">
         <div id="resourceTemplate">
           <Alerts errorKey={resourceEditErrorKey(this.props.resourceKey)} />
+          <Alerts errorKey={'newresource'} />
```
Comment (justinlittman) [inline src/components/editor/ResourceTemplate.jsx]: Can you not hard code this and import from ResourceList?

**Pull request merged**